components/color-picker/components/ColorTrigger.tsx

```diff
@@ -40,6 +42,9 @@ const ColorTrigger = forwardRef<HTMLDivElement, colorTriggerProps>((props, ref)
       {...rest}
     >
       {containerNode}
+      {typeof textRender === 'function' && (
```

Comment:
这个在 ts 中已经约束了函数，这里不需要判断类型了吧

```suggestion
      {textRender && (
```

Reply:
问题也不大，因为后面也是加了可选链的，也不会报错，可以去掉。

Reply:
兜底也可以，可以保证 js 不会报错

Reply:
我的想法是这种情况反而不需要兜底，如果用户传了不是函数，就让 js 把报错抛出去，兜底的话用户反而看不到错误，不知道为啥这里不显示 dom

Reply:
如果值隐式转换成`boolean` ，可能会多一个空节点。